lib/src/Uncertainty/Algorithm/MetaModel/Kriging/KrigingAlgorithm.cxx

```diff
@@ -75,7 +75,9 @@ KrigingAlgorithm::KrigingAlgorithm(const NumericalSample & inputSample,
   , covarianceCholeskyFactor_()
   , covarianceCholeskyFactorHMatrix_()
 {
-  glmAlgo_ = GeneralizedLinearModelAlgorithm(inputSample, outputSample, covarianceModel, basis, normalize, keepCholeskyFactor);
+  // Here we must force the GLMAlgo to keep the Cholesky factor as it is mandatory
+  // for the interpolation part
+  glmAlgo_ = GeneralizedLinearModelAlgorithm(inputSample, outputSample, covarianceModel, basis, normalize, true);
```

Comment:
This call enforce the keepCholeskyFactor flag to true when calling GLMAlgo, which is mandatory to perform the interpolation step.